test/test_inflation.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from fastapi.testclient import TestClient
+from main import app
+import pytest
+from fastapi.testclient import TestClient
+from main import app
+
+client = TestClient(app)
+
+
+def test_calculate_inflation_with_mock_data(monkeypatch):
+    # Mock the mongo_collection.find method to return sample data
+    def mock_find(query):
+        return [
+            {"price": 10},
+            {"price": 20},
+            {"price": 15}
+        ]
+
+    monkeypatch.setattr(app.state.mongo_collection, "find", mock_find)
```

Comment:
You should mock `pymongo.collections.MongoCollection` (path may be wrong) but not app's attributes